cql3/statements/index_target.cc

```diff
@@ -35,11 +36,21 @@ sstring index_target::as_string() const {
         }
 
         sstring operator()(const ::shared_ptr<column_identifier>& column) const {
-            return column->to_string();
+            sstring postfix;
+            switch (target->type) {
+                case index_target::target_type::keys:
```

Comment:
nitpick: add `[[fallthrough]];` for each fallthrough case

Reply:
It turns out that all this code was wrong anyway ;-) This "as_string()" function was just used in three places, all three expected to get just the column name - **not** the column name with "_" + type tacked at the end. I removed it in my new version of this PR (in a separate patch at the end). So resolving this conversation.